src/equations.jl

```diff
@@ -1,190 +1,128 @@
 # Tracing equations.
 
-"""
-	 trace!(dy, y, p::TPTuple, t)
-	 trace!(dy, y, p::FullTPTuple, t)
-
-ODE equations for charged particle moving in static EM field with in-place form.
-
-ODE equations for charged particle moving in static EM field and external force field with in-place form.
-"""
-function trace!(dy, y, p::TPTuple, t)
-	q2m, Efunc, Bfunc = p
-
-	v = y[SA[4:6...]]
-	E = Efunc(y, t)
-	B = Bfunc(y, t)
-
-	dy[1:3] = v
-	dy[4:6] = SVector{3}(q2m * (v × B + E))
+function get_dv(u, v, p, t)
```

Comment:
Maybe rename `u` to `x` here to make it clear that the first argument is ordinary space coordinate?